app/components/notification_message_component.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="govuk-notification-banner govuk-notification-banner--<%= notification_type %>" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
```

Comment:
can we also include the role here as well?
```
role="alert"
```

Reply:
I completely missed this, thanks for pointing it out.

Added a bit of logic behind this so we render `role="alert"` or `role="region"` as mentioned on the GDS component page system. More specific information can be found on the relevant commit message.